presto-native-execution/src/test/java/com/facebook/presto/nativetests/TestNativeOrderByQueries.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativetests;
+
+import com.facebook.presto.nativeworker.PrestoNativeQueryRunnerUtils;
+import com.facebook.presto.testing.ExpectedQueryRunner;
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.AbstractTestOrderByQueries;
+import org.testng.annotations.Ignore;
+import org.testng.annotations.Test;
+
+public class TestNativeOrderByQueries
+        extends AbstractTestOrderByQueries
+{
+    @Override
+    protected QueryRunner createQueryRunner() throws Exception
+    {
+        return PrestoNativeQueryRunnerUtils.createNativeQueryRunner(false);
+    }
+
+    @Override
+    protected ExpectedQueryRunner createExpectedQueryRunner() throws Exception
+    {
+        return PrestoNativeQueryRunnerUtils.createJavaQueryRunner();
+    }
+
+    @Override
+    @Test
+    public void testOrderByWithOutputColumnReference()
```

Comment:
i) Lets not rewrite SQL -- use assertQueryFails with the error message and we fix them over time.

Reply:
Thanks for the suggestion, the assertQuery rewrites are reverted. assertQueryFails expects the Sql query and the error message as the parameters. The failing queries all have expectedResult (in Sql) as a parameter, which cannot be added to assertQueryFails. 
The failing tests have been moved to a separate disabled testcase. Could you please confirm if this is fine?